lms/templates/calculator/toggle_calculator.html

```diff
@@ -1,144 +1,157 @@
-                    <td>20%<br />
-                      20c<br />
-                      418T
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Basic functions")}</th>
-                    <td>abs, exp, fact or factorial, ln, log2, log10, sqrt</td>
-                    <td>abs(x+y)<br />
-                      sqrt(x^2-y)
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Trigonometric functions")}</th>
-                    <td>sin, cos, tan, sec, csc, cot<br />
-                      arcsin, sinh, arcsinh, etc.<br />
-                    </td>
-                    <td>sin(4x+y)<br />
-                      arccsch(4x+y)
-                    </td>
-                    <td></td>
-                  </tr>
-                  <tr>
-                    ## Translators: Please see http://en.wikipedia.org/wiki/Scientific_notation
-                    <th scope="row">${_("Scientific notation")}</th>
-                    ## Translators: 10^ is a mathematical symbol. Please do not translate.
-                    <td>${_("10^ and the exponent")}</td>
-                    <td>10^-9</td>
-                  </tr>
-                  <tr>
-                    ## Translators: this is part of scientific notation. Please see http://en.wikipedia.org/wiki/Scientific_notation#E_notation
-                    <th scope="row">${_("e notation")}</th>
-                    ## Translators: 1e is a mathematical symbol. Please do not translate.
-                    <td>${_("1e and the exponent")}</td>
-                    <td>1e-9</td>
-                  </tr>
-                </tbody>
-              </table>
-            </li>
-          </ul>
-        </div>
-      </div>
+    <button title="${_('Open Calculator')}" aria-controls="calculator_wrapper" aria-expanded="false" class="calc utility-control-button">
+        <span class="icon fa fa-calculator" aria-hidden="true"></span>
+    </button>
+    
+    <div id="calculator_wrapper">
+        <form id="calculator">
+            <div class="input-wrapper">
+                <label for="calculator_input" class="label-calc-input sr">${_('Enter equation')}</label>
+                <input type="text" id="calculator_input" title="${_('Calculator Input Field')}" aria-describedby="calculator_output" />
+                
+                <div class="help-wrapper">
+                    <button type="button" title="${_('Hints')}" id="calculator_hint" aria-haspopup="true" aria-expanded="false" aria-controls="calculator_input_help"></button>
+                    <ul id="calculator_input_help" class="help" aria-hidden="true">
+                        <li class="hint-item" id="hint-moreinfo">
+                            <p class="sr" id="hint-instructions">${_('Use the arrow keys to navigate the tips or use the tab key to return to the calculator')}</p>
```

Comment:
I moved this "hint" inside of the "hint panel" because it was previously being announced prematurely. There is a lot of work we _could_ do to the calculator, but my aim is to fix the one tiny thing. This has blown up beyond the initial scope.

Tomorrow I will get anything out of order, in proper order and update the sandbox for a final review. Note the initial scope of work was to simply change an invalid ARIA attribute. In my quest to "fix all the things!" I've let this get way out of control.

In short, I'll verify tomorrow that things work correctly, then have you guys review the safe template updates (understandably necessary) and the ARIA change. Everything else should work as it has before, funky or not.